config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -21,6 +21,25 @@ postsubmits:
       - name: github
         secret:
           secretName: oauth-token
+  kubernetes/test-infra:
+  - name: post-test-infra-push-prow
+    cluster: test-infra-trusted
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-bazel:v20181002-996397d
+        command:
+        - prow/bump.sh
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
probably add:
```
- name: USER
  value: prow
```
the ssh preset does this
/shrug

Reply:
I'm not sshing

Reply:
right, but the $USER will be in the outputs?

Reply:
I'm not saying to use the ssh preset, just saying that we have precedent for setting this and it will probably make the output less surprising.

Reply:
doesn't seem required here?

Reply:
wasn't it tagging things as root?

Reply:
we should probably just remove the $USER-latest tag. I don't think anyone has ever used this. Given that everyone should be using the unique `YYYYMMDD-deadbeef` tag I don't really care what the value of $USER is and I don't want to increase the job complexity to make it a "better value" given that no one is going to use that tag.

Reply:
Aka whenever we push an image we push it using three tags:
* `latest`
* `$USER-latest` (or maybe `latest-$USER` I forget)
* `YYYYMMDD-deadbeef`

The last one is what we want to advertise.